<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QDialog" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="ProductsList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Products</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Product Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditcategory">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonApplyFilter">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply Filter</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>471</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddtoCart">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Add To Cart</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditbrand">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDiscount">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Discount</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelbrand">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>brand</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Shopping Cart</string>
    </attribute>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>11</y>
       <width>471</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConfirm">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>270</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total Amount = &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
